### ISSUE TITLE
Alt text for non-keynote images 

### ISSUE THREAD

**Issue by ldecicco-USGS** [USGS-VIZLAB/great-lakes-microplastics#229]
When the keynote images are hovered over, the alt text shows up. This doesn't happen on the static images. Is there some other tag we should be using? Or is it fine as is? 

```
<img src="images/Piper.jpg" alt="Piping Plover. Photo by Jim Hudgins/USFWS."/>
```

**Comment by mwernimont** [comment]
The BG's have a title, which pop up with mouse overs. Alt text is more for text readers. If you want the hover over for the other images just add a title to it. `<img src="images/Piper.jpg" alt="Piping Plover" title="What you want to appear over hover"/>`

**Comment by mwernimont** [comment]
But that said titles aren't a thing you absolutely need, more if you want it. So if we add it or not no harm or foul either way

**Comment by ldecicco-USGS** [comment]
Cool, good to know. I'll add

**Comment by ldecicco-USGS** [comment]
What should the difference between the two be in terms of content? Can they be the same?

**Comment by mwernimont** [comment]
They can be exactly the same.

**Comment by mwernimont** [comment]
No biggie

**Comment by mwernimont** [comment]
whoops, well this website says this "Image title (and the element name speaks for itself) should provide additional information and follow the rules of a regular title: it should be relevant, short, catchy, and concise (A title “offers advisory information about the element for which it is set.“). In FireFox and Opera it pops up when you hover over an image::

Alt text is meant to be an alternative information source for those people who have chosen to disable images in their browsers and those user agents that are simply unable to “see” the images. It should describe what the image is about and get those visitors interested to see it. (Like the picture to the right)

So title should be shorter and less descriptive
